R/pkg/R/SQLContext.R

```diff
@@ -330,6 +330,30 @@ jsonRDD <- function(sqlContext, rdd, schema = NULL, samplingRatio = 1.0) {
   }
 }
 
+#' Create a SparkDataFrame from a ORC file.
+#'
+#' Loads a ORC file, returning the result as a SparkDataFrame.
+#'
+#' @param path Path of file to read.
+#' @return SparkDataFrame
+#' @rdname read.orc
+#' @export
+#' @name read.orc
+#' @method read.orc default
+#' @note read.orc since 2.0.0
+read.orc.default <- function(path) {
```

Comment:
Since read.orc is a new API method, this is not needed for backward compatibility?